src/test/java/com/github/fishio/logging/TestTxtFileHandler.java

```diff
@@ -76,9 +76,15 @@ public void testSetGetBW() {
 	
 	/**
 	 * Test if output calls correct method.
+	 * 
+	 * @throws IOException
+	 * 		we verify that write was called on our mocked buffered writer,
+	 * 		but the exception has to be caught. Since we are only verifying
+	 * 		and not actually calling the method, this exception cannot occur.
+	 * 		This throws is here to make the code cleaner.
 	 */
 	@Test
-	public void testOutput() {
```

Comment:
@Diocruel That will be fixed in PR #217, commit 0ee147c. This PR still makes PMD happier, by removing one warning a few lines below this one.

<layout